redis-sink/src/main/resources/application.yml

```diff
@@ -0,0 +1,12 @@
+server:
+  port: 8081
+spring:
+  cloud:
+    stream:
+      bindings:
+        input: testtock
```

Comment:
I think we should standardize all OOTB modules to have some vanilla string here, something like `demostream`